Makefile

```diff
@@ -431,6 +431,7 @@ kind-clustermesh: ## Create two kind clusters for clustermesh development.
 kind-clustermesh-down: ## Destroy kind clusters for clustermesh development.
 	kind delete clusters clustermesh1
 	kind delete clusters clustermesh2
+	docker network rm kind-cilium
```

Comment:
could we change kind-down.sh to take clusters to delete as optional argument to not duplicate the logic and for symmetry? Then this could become just `kind-down.sh clustermesh1 clustermesh2`. WDYT?

Reply:
Done